parser/Module.md

```diff
@@ -109,3 +109,111 @@ public function main() {
 ```
 
 *Note:* `parse` function returns `anydata` type when success, and it needs to be cast to the relevant FHIR Resource type.
+
+**03. Parse and validation with a valide FHIR data**
+
+In this approach the parser will attempt to parse and apply to the validation the given FHIR resource payload to given resource type profile.
```

Comment:
Shall we update this as, `In this approach, the parser will attempt to parse the given FHIR resource payload and apply validation against the specified resource type profile.`

Reply:
Updated